index.html

```diff
@@ -259,10 +259,200 @@ <h1>Architecture</h1>
                 alt="Discovery process overview" />
             <figcaption>Discovery process overview</figcaption>
         </figure>
-        <p class="ednote" title="Discovery Architecture Overview">
-        To do: an overview of the two-phase approach and its purpose, which is to support
-        controlled and authenticated access to metadata by authorized users only.
+	<p>
+	The WoT discovery process is designed using a two-phase approach.
+	In the first phase, one or more of a set of "Introduction" mechanisms may be used to 
+	generate a set of candidate URLs.  These URLs do not themselves contain metadata,
+	but are used in the second stage, "Exploration", to actually fetch metadata.
+	The intention is that Introductions are relatively open and can support any
+	mechanism that can return a URL, including simply entering URLs directly.
+	Introductions, however, do not include any security or privacy controls and
+	so should not provide metadata directly.  Instead, the provided URLs are followed
+	to "Exploration" mechanisms that will actually provide metadata but allow for
+	suitable security and privacy controls.
         </p>
+	<p>
+	Note that the discovery process can produce a <em>set</em> of URLs following 
+	its Introduction phase, even if only one Introduction mechanism is used,
+	and the final output after the Exploration phase
+	can be a <em>set</em> of Thing Descriptions.
```

Comment:
Same as last comment. If the process is to discover one Thing, why is there a "set of URLs" after Intro, even if only one mechanism is used?

Reply:
Some Intro mechanisms can return multiple results.  Will discuss today.